resources/install/example.sql

```diff
@@ -19,3 +19,14 @@ CREATE TABLE IF NOT EXISTS `activities_activities` (
     KEY `row` (`row`),
     KEY `ip` (`ip`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS `activities_resources` (
```

Comment:
Thanks for the schema link. resource_id  sounds good. The primary key would then become UUID. On TEXTman, what's the revision column for? I'm using created_on for sorting in chronological order, wouldn't that be enough?